psoc/grobot_workspace/grobot_module_controller.cydsn/dht11.c

```diff
@@ -0,0 +1,234 @@
+//#include <project.h>
+
+#include "dht11.h"
+#include <stdint.h>
+#include <dht.h>
+
+#define null 0
+
+void delay(void);
+void unpack_dht_data(void);
+
+uint8_t check;
+uint8_t decode_flag;
+uint8_t decoded_temperature_data;
+uint8_t decoded_humidity_data;
+
+int temperature_values[8];
+int humidity_values[8];
+int diff_count_values[44]={0};
```

Comment:
Quick style thing: Prefer spaces around equal signs, e.g. ```int diff_count_values[44] = 0```.
(You also only need the curly braces when you're initializing an entire array, not when you're just assigning an element.)

Incidentally, this goes for all binary operators, pretty much.